admin_manual/release_notes.rst

```diff
@@ -0,0 +1,266 @@
+================================
+ownCloud |version| Release Notes
+================================
+
+Recommended Setup for Running ownCloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ownCloud runs on multiple operating systems and HTTP servers. For best 
+performance, stability, support, and full functionality we recommend:
+
+* Red Hat Enterprise Linux 7
+* MySQL/MariaDB
+* PHP 5.4 +
+* Apache 2.4
+
+Supported Platforms
+^^^^^^^^^^^^^^^^^^^
+
+* Hypervisors: Hyper-V, VMware ESX, Xen, KVM
+* Server: Windows 2008R2; 2012, Linux (Debian 6, RHEL / Centos 6 and 7, Ubuntu 
+  12 and 14, SLES 11, Univention Corporate Server 3.x)
+* Databases: MySQL 5.x; Microsoft SQL Server 2008R2; SQL Server 2012 R2; Oracle 
+  11g
+* Desktop: Windows XP, Windows 7, Mac OS X 10.7+ (64-bit only), Linux (CentOS 
+  6, Ubuntu 10.04 and 11.04+, Fedora 16+, openSUSE 11.4+)
```

Comment:
Updated list from @MTRichards:

Windows 7+
Windows XP SP3 (EoL Q2 2015) <-extended support ended last year in april, so
if we support it now, we can phase it out by June.
Mac OS X 10.7+ (64-bit only)
CentOS 6.5, 7
Ubuntu 12.04 LTS, 14.04 LTS, 14.10
Fedora 20, 21
openSUSE 12.3, 13